pandas/tests/groupby/test_groupby.py

```diff
@@ -2954,3 +2954,38 @@ def test_groupby_dropna_with_nunique_unique():
     )
 
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize("as_index_flag", [False])
```

Comment:
You can just inline `as_index=False` in the `groupby` call below

Reply:
Got it, I've just updated the PR!